tools/osv-linter/internal/pkgchecker/ecosystems.go

```diff
@@ -0,0 +1,240 @@
+package pkgchecker
```

Comment:
@cuixq can you please also review this file as @andrewpollock requested?

Reply:
done! (sorry about missing the request)

Reply:
All good, thank you!